spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -318,6 +318,7 @@
 "Nokia Open Source License (NOKOS)": Nokia
 "ODbL 1.0": ODbL-1.0
 "ODbL v1.0": ODbL-1.0
+"OSI Approved :: BSD License":  BSD-3-Clause
```

Comment:
We [strip](https://github.com/oss-review-toolkit/ort/blob/122331e69d098820958869984a10023208990a7a/analyzer/src/main/kotlin/managers/Pip.kt#L493) the "OSI Approved" part, so this should only map `BSD License`.

Reply:
But for that we actually already have a mapping.

Reply:
We strip the "OSI Approved ::" if it occurs in the classifiers see [1] but in this case it happens in the license. Should we fix `Pip.kt`? or add a mapping?

[1] https://github.com/oss-review-toolkit/ort/blob/d8bffdce788696a3e0985d937c9b19f9f7f200d4/analyzer/src/main/kotlin/managers/Pip.kt#L493

Reply:
AFAIK the double-colon syntax is reserved for classifiers, so IMO this package simply has broken license meta-data then. I'd say "OSI Approved :: BSD License" is not a *common* declared name for `BSD-3-Clause`, so IMO this should not be addressed by a mapping but by a curation for the package.

Reply:
Agreed dropped the "OSI Approved :: BSD License" commit from this PR